google/auth/external_account.py

```diff
@@ -40,6 +41,7 @@
 from google.auth import exceptions
 from google.auth import impersonated_credentials
 from google.auth import metrics
+from google.auth.transport.requests import _MutualTlsAdapter
```

Comment:
Is it possible to keep this self-contained to the transport module? (I don't see this being used)